lib/rules/prefer-node-protocol.js

```diff
@@ -124,7 +124,7 @@ module.exports = {
                             !isBuiltinModule(value) ||
                             !isBuiltinModule(`node:${value}`)
                         ) {
-                            return
```

Comment:
Do you think the same issue occurs at [lib/rules/prefer-node-protocol.js:113](https://github.com/coderaiser/eslint-plugin-n/blob/fix/prefer-node-protocol-not-first/lib/rules/prefer-node-protocol.js#L113)

Reply:
You right, better to handle this in the same way https://github.com/eslint-community/eslint-plugin-n/pull/206